libbpf-cargo/src/gen.rs

```diff
@@ -741,6 +733,16 @@ fn gen_skel_contents(_debug: bool, raw_obj_name: &str, obj_file_path: &Path) ->
     gen_skel_attach(&mut skel, object, &obj_name)?;
     writeln!(skel, "}}")?;
 
+    // Coerce to &[u8] just to be safe, as we'll be using debug formatting
+    let bytes: &[u8] = &*mmap;
+    write!(
+        skel,
+        r#"
+        const DATA: &[u8] = &{:?};
```

Comment:
it's probably a good idea to split array into multiple lines, if possible. Single very long line will cause troubles for a lot of editors, making it really hard to look at the generated skeleton file. So please consider doing that.

Reply:
Rustfmt breaks it up into 100 char wide lines already